test/test_suite.cpp

```diff
@@ -283,7 +283,7 @@ struct Approach
     csubstr casename;
     csubstr filename;
     ProcLevel levels[NLEVELS] = {};
-    AllowedFailure allowed_failure = {};
+    AllowedFailure allowed_failure;
```

Comment:
Same question as above. Is this correctly zero-initialized by g++4.8?

Reply:
It is default initialized, but all members are objects (csubstr) which will in turn be default init, so it should be ok for that one.

Reply:
Based on the thread above, let's keep the `{}` also in here, and just suppress 4.8 warnings if needed.